lib/Cache/CacheClearer.php

```diff
@@ -68,6 +68,6 @@ public function clear(string $buildDir) : array
      */
     protected function glob(string $pattern) : array
     {
-        return glob($pattern);
+        return array_filter((array) glob($pattern));
```

Comment:
A phpstan error tells that `glob()` seems to be able to return `false` and also `array<int, false>`. This is why I created this strange filter-typecast construct.